### DISCUSSION TITLE
Made advanced exporter?

### DISCUSSION THREAD

**Discussion by rahman-nero** [Laravel-Backpack/community-forum#784] [Show and Tell]
Currently we are using backpack 5.4 and there is one problem with list operations (I don't know it has been fixed or not). When I try to export some data from table, it exports from client side and only the part that is visible, so it's the problem. 

For example, If we have more than 100 rows, all rows that exceeds 100 won't be exported. So for solution I wrote this package (https://github.com/rahman-nero/backpack-export) that export data on backend (in a job). 

P.S I wrote the package for 5-7 hours and also there is Cyrillic comments (cuz first it was package for only our company, but at the end I decided to publish it). I will be working on it further and if you have some ideas, make sure to leave there :). 

So you may wonder why I wrote this?. Answer: I hope I will help someone with the issue of exporting

**Comment by rahman-nero** [comment]
Oh, It was here (https://github.com/Laravel-Backpack/community-forum/discussions/660). I didn't know you had it planned :)

**Comment by jcastroa87** [comment]
Hi @rahman-nero 

thank you very much for your contribution to the community, as you discovered we are working on the exporter as part of our roadmap and one of our developers will take a look at your repository to check if it is compatible with the progress in development we have so far. Thanks again.

Cheers.

**Comment by jcastroa87** [comment]
Hi @pxpm can you check this repo to let @rahman-nero know if we can use some of this code.

Thanks.

**Comment by Bad-Y** [comment]
Hi everyone,

There is this package to export : https://github.com/redsquirrelstudio/laravel-backpack-export-operation
Also this one to import : https://github.com/redsquirrelstudio/laravel-backpack-import-operation

You should check them.
